src/channels/twilio/renderers/choices.ts

```diff
@@ -3,7 +3,7 @@ import { ChoicesRenderer } from '../../base/renderers/choices'
 import { TwilioContext } from '../context'
 
 export class TwilioChoicesRenderer extends ChoicesRenderer {
-  renderChoice(context: TwilioContext, payload: ChoiceContent) {
+  async renderChoice(context: TwilioContext, payload: ChoiceContent) {
```

Comment:
Since `prepareIndexResponse` needs to be async (kvs.set is async), the renderer must also be async, meaning that we might have to handle this in the base conduit. What do you think?